src/app/shared/components/header/header.component.html

```diff
@@ -1,7 +1,8 @@
 <div class="app-header">
     <div class="app-header-container container">
         <app-logo class="mr-auto"></app-logo>
-        <div class="button button--transparent"><span class="icon icon-user"></span>{{login}}</div>
-        <div class="button button--transparent"><span class="icon icon-exit"></span>Log off</div>
+        <div *ngIf="isAuthenticated" class="button button--transparent" (click)="onUserInfo($event)"><span class="icon icon-user"></span>{{login}}</div>
+        <div *ngIf="isAuthenticated" class="button button--transparent" (click)="onLogoff($event)"><span class="icon icon-exit"></span>Log off</div>
```

Comment:
I suggest optimizing this part by using ng-container with one subscription.
In that case, we reduce subscriptions from two to one :)

something like that

```
<ng-container *ngIf="isAuthenticated">
  <div> ...
  <div> ...
```

Reply:
done